includes/apple-exporter/components/class-quote.php

```diff
@@ -243,36 +438,25 @@ private function _set_blockquote_style() {
 	 */
 	private function _set_pullquote_anchor() {
 		$this->set_anchor_position( Component::ANCHOR_AUTO );
-		$this->json['anchor'] = array(
-			'targetComponentIdentifier' => 'pullquoteAnchor',
-			'originAnchorPosition' => 'top',
-			'targetAnchorPosition' => 'top',
-			'rangeStart' => 0,
-			'rangeLength' => 10,
-		);
 	}
 
 	/**
 	 * Set the border for a pullquote.
 	 *
+	 * @param array $values
+	 * @return array
```

Comment:
Actually, this is correct. You identified a bug in the code which I've addressed.